README.md

```diff
@@ -221,6 +223,14 @@ To use ChatGPT Code Review, add an `OPENAI_API_KEY` environment variable with an
 
 ![image](https://user-images.githubusercontent.com/28964599/233685071-e1371edf-6359-41c3-a989-335d6ee09cb7.png)
 
+## 💻 Environment Variables handling
+
+[T3 Env](https://env.t3.gg/) is a library that provides environmental variables checking at build time, type validation and transforming. It ensures that your application is using the correct environment variables and their values are of the expected type. You’ll never again struggle with runtime errors caused by incorrect environment variable usage.
```

Comment:
could we add a note about the location of the config file here?

Reply:
good point, added it along with example